Homework9-1medium/change-button.js

```diff
@@ -0,0 +1,18 @@
+
+const button= document.getElementById("btn")
+button.addEventListener("click",change)
+
+
+var clicks = 0;
+function change() {
+  clicks += 1;
+   if(clicks==2)
```

Comment:
Congratulations on hacking the problem I mentioned above 👏 . But this solution merely contains the problem, it does not fixes it.By fixing the above though you will have to modify your algorithm in this function a bit.